src/model/Series.ts

```diff
@@ -558,6 +569,14 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
         }
 
         const data = this.getData(dataType);
+        if (data.getItemModel<StatesOptionMixin<unknown, unknown>>(dataIndex).get(['select', 'disabled'])) {
```

Comment:
It's better to do this check after `selectedMap` check because `getItemModel` and `Model#get` is much more expensive than simply object access in `selectedMap`. We need to avoid this call as much as possible.

Reply:
If we put `selectedMap` at front,  it will select the one whose `selected` is disable.

Reply:
How this will happen? disabled is still checked. But it's only checked when it's in the selectedMap or selectedMap is 'all'